website/versioned_docs/version-5.x/ecosystem-angular.md

```diff
@@ -499,6 +499,37 @@ Then use it in your template:
 ```html
 <img [src]="'yoshi.png' | assetUrl" />
 ```
+##### Option 3
+
+upload via sass; This demonstration was done with the angular 8 and 9 versions, example: 
+
+```sass
+.body-row {
+    background: url("/assets/images/person.jpg") no-repeat right bottom
+}
+```
+To be able to show your images using Sass URL we have to do some configuration in our angular.json file, should go to the attribute
+**architect > build > options**and add the attribute  **"rebaseRootRelativeCssUrls"** value "true".
+Now inside the assets attribute (which is an array) you should add a slash **"/"** to the value of the **"ouput"** attribute which outputs its external styles (this is optional); example
+
+ ```json
```

Comment:
The `...` in the json makes the json invalid, which shows syntax highlighting errors in the markdown:

![image](https://user-images.githubusercontent.com/5524384/84704083-8395df00-af16-11ea-837e-722521765fd0.png)

Please do the following:

1. Remove the `...`
2. Switch from tabs to spaces.

Reply:
You are right, I put those ellipsis hinting that there may be more attributes inside the json

Reply:
I already removed the ellipsis